page-data.php

```diff
@@ -51,8 +52,9 @@
       ],
       '#js-TJIChartView',  // Element to build the charts into
       '#secondary',  // Element to put the filter panel inside
-      '<div class="col-sm-12 col-lg-6" />',  // Wrapper for chart canvases
-      '<div class="col-sm-12 record-count">{count} records</div>',  // Template for record count element
+      '<div class="col-xs-12 col-sm-12 col-md-6 col-xlg-4 tji-chart" />',  // Wrapper for chart canvases
```

Comment:
I don't think 'col-xlg-4' is a thing -- can we remove this so it's not confusing.